app/LLM_extractions/validations.py

```diff
@@ -0,0 +1,62 @@
+from typing import Optional, Any
+
+
+def validate_age_order(filtered_ordered_response: dict) -> Optional[Any]:
+    """
+    Validate that min_age is not greater than max_age.
+    If validation fails, prompt user for confirmation to swap ages.
+
+    Args:
+        filtered_ordered_response (dict): Filtered dictionary of extracted parameters.
+
+    Returns:
+        dict or str: Validated response with swapped ages or error message.
+                    Returns None if no action is required.
+    """
+
+    if (
+        "min_age" in filtered_ordered_response
+        and "max_age" in filtered_ordered_response
+    ):
+        min_age = int(filtered_ordered_response["min_age"])
+        max_age = int(filtered_ordered_response["max_age"])
+
+        if min_age > max_age:
+            confirmation = input(
+                f"Sorry but minimum age ({min_age}) can’t be bigger than maximum age ({max_age}). Would you like to see the result for min age {max_age} and max age {min_age}? If yes, please confirm (yes/no): "
+            ).lower()
+
+            if confirmation == "yes":
```

Comment:
Generally a good fail safe to have in place

```suggestion
            if confirmation.lower() == "yes":
```